lib/wallet/walletdb.js

```diff
@@ -196,6 +196,12 @@ class WalletDB extends EventEmitter {
       this.state.height,
       this.state.startHeight);
 
+    const clientInfo = await this.client.getInfo();
+    this.chainInfo = {
```

Comment:
There is no null value for `chainInfo` in the constructor

Reply:
ah thanks 👍

Reply:
Should it be `null` or an expected default value like
```
{
  spv: false
  prune: false
}
```

Reply:
I think it should at least be an object because we do not want to accidentally cause a problem where a property on `null` is referenced

I think the expected default value makes sense in this case

Reply:
went with `{}` for null value in case we want to expand, it's one less place to have to update. and it'll return `undefined` instead of throwing for requested properties

Reply:
👍 good point